SmartDeviceLink/SDLMutableDataQueue.h

```diff
@@ -0,0 +1,38 @@
+//
+//  SDLMutableDataQueue.h
+//  
+//
+//  Created by David Switzer on 3/2/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface SDLMutableDataQueue : NSObject
```

Comment:
What's the purpose of this class handling `NSMutableData` instead of just `NSData`